tests/test_hep.py

```diff
@@ -131,65 +131,59 @@ def test_classification_number(marcxml_to_json, json_to_marc):
+        assert (val['source'] ==
                 json_to_marc['084'][index]['9'])
-        assert (val['standard'],
+        assert (val['standard'] ==
                 json_to_marc['084'][index]['2'])
 
 
 def test_authors(marcxml_to_json, json_to_marc):
     """Test if authors are created correctly."""
-    assert (marcxml_to_json['authors'][0]['full_name'],
+    assert (marcxml_to_json['authors'][0]['full_name'] ==
             json_to_marc['100']['a'])
-    assert (marcxml_to_json['authors'][0]['role'],
+    assert (marcxml_to_json['authors'][0]['role'] ==
             json_to_marc['100']['e'])
-    assert (marcxml_to_json['authors']
-            [0]['alternative_name'],
+    assert (marcxml_to_json['authors'][0]['alternative_name'] ==
             json_to_marc['100']['q'])
-    assert (marcxml_to_json['authors'][0]['inspire_id'],
+    assert (marcxml_to_json['authors'][0]['inspire_id'] ==
             json_to_marc['100']['i'])
-    assert (marcxml_to_json['authors'][0]['orcid'],
+    assert (marcxml_to_json['authors'][0]['orcid'] ==
             json_to_marc['100']['j'])
-    assert (marcxml_to_json['authors'][0]['email'],
+    assert (marcxml_to_json['authors'][0]['email'] ==
             json_to_marc['100']['m'])
-    assert (marcxml_to_json['authors'][0]['affiliations'][0]
-            ['value'],
+    assert (marcxml_to_json['authors'][0]['affiliations'][0]['value'] ==
             json_to_marc['100']['u'][0])
-    assert (marcxml_to_json['authors'][0]['recid'],
+    assert (marcxml_to_json['authors'][0]['recid'] ==
             json_to_marc['100']['x'])
-    assert (marcxml_to_json['authors'][0]
-            ['curated_relation'],
+    assert (marcxml_to_json['authors'][0]['curated_relation'] ==
             json_to_marc['100']['y'])
 
 
 def test_corporate_author(marcxml_to_json, json_to_marc):
     """Test if corporate_author is created correctly."""
-    assert (marcxml_to_json['corporate_author'][0],
+    assert (marcxml_to_json['corporate_author'][0] ==
             json_to_marc['110'][0]['a'])
 
 
 def test_title_variation(marcxml_to_json, json_to_marc):
     """Test if title_variation is created correctly."""
-    assert (marcxml_to_json['title_variation']
-            [0],
+    assert (marcxml_to_json['title_variation'][0] ==
             json_to_marc['210'][0]['a'])
 
 
 def test_title_translation(marcxml_to_json, json_to_marc):
     """Test if title_translation is created correctly."""
-    assert (marcxml_to_json['title_translation']
-            [0]['title'],
+    assert (marcxml_to_json['title_translation'][0]['title'] ==
             json_to_marc['242'][0]['a'])
-    assert (marcxml_to_json['title_translation']
-            [0]['subtitle'],
+    assert (marcxml_to_json['title_translation'][0]['subtitle'] ==
             json_to_marc['242'][0]['b'])
 
 
 def test_title(marcxml_to_json, json_to_marc):
     """Test if title is created correctly."""
-    assert (marcxml_to_json['titles'][1]['title'],
```

Comment:
The arXiv title actually goes in another `json_to_marc` field `246`, which is why only one of the titles are checked. And yeah, `[0]` index on both.